deployments/helm/nvidia-device-plugin/templates/_helpers.tpl

```diff
@@ -269,3 +269,20 @@ We convert this to JSON so that it can be included and converted to an object us
 {{- $_ := set $options "addMigMonitorDevices" ( ne ( (include "nvidia-device-plugin.allPossibleMigStrategiesAreNone" . ) | trim ) "true" )  -}}
 {{- mustToJson $options -}}
 {{- end -}}
+
+{{/*
+ImexNodesConfigFile path
```

Comment:
@elezar this could help us to always prepend `nvidiaDriverRoot`

Reply:
I don't think it should be so specific to IMEX -- it should be a helper that does the equivalent of  https://pkg.go.dev/path/filepath#Join (i.e. concatenate all of its arguments as path elements of a full path). With that you can call it as `{{ pathjoin .Values.nvidiaDriverRoot /etc/nvidia-imex }}` and it will "do the right thing"

I think this is why @elezar asked if something like this already exists, since it very well may

Reply:
I wonder if `clean` would do what we want:
https://helm.sh/docs/chart_template_guide/function_list/#clean

It only mentions the relative path thing, but maybe it would also remove redundant `/`s

Reply:
I created a helper function to do just the same that `filepath.Join` does